README.md

```diff
@@ -8,16 +8,16 @@ This is a **work-in-progress** decompilation of LEGO Island (Version 1.1, Englis
 
 <img src="https://legoisland.org/progress/ISLEPROGRESS.SVG" width="50%"><img src="https://legoisland.org/progress/LEGO1PROGRESS.SVG" width="50%">
 
-Currently, `ISLE.EXE` is completely decompiled and behaves identically to the original. A handful of stubborn instructions are not yet matching; however, we anticipate they will as more of the overall codebase is implemented.
+Currently, `ISLE.EXE` is completely decompiled and is functionally identical to the original. Due to various complexities with regard to the compiler, it is not a byte-for-byte match of the original executable. We remain hopeful that this can be resolved at some point.
```

Comment:
I think I'd merge these two paragraphs. Everything that is said applies equally to both modules imo

Reply:
Good idea!

Reply:
I merged the ISLE and LEGO1 statues into one paragraph. I kept the second paragraph but kept it limited to the note about accuracy only, just because I feel that this breaks up the reading flow better.